src/actinia_core/rest/ephemeral_processing.py

```diff
@@ -1676,10 +1676,20 @@ def _parse_module_outputs(self):
                         key, value = row.split(delimiter, 1)
                         result[key.strip()] = value.strip()
                 elif "json" in format:
+                    result = None
                     try:
-                        result = json.loads(stdout)
+                        result = {i[0]: i[1] for i in [
+                            entry.split('=') for entry in
```

Comment:
```suggestion
                            entry.split(delimiter, 1) for entry in
```
If i see it correctly, you did not use the `delimiter`.